api/jsonapi.go

```diff
@@ -2266,7 +2265,7 @@ func (i *jsonAPIHandler) POSTReleaseEscrow(w http.ResponseWriter, r *http.Reques
 
 func (i *jsonAPIHandler) POSTSignMessage(w http.ResponseWriter, r *http.Request) {
 	type plaintext struct {
-		Content string `json:"content"`
+		Content []byte `json:"content"`
```

Comment:
Can a client easily post a byte array?

Reply:
The client won't need to change anything, it just marshals the json value into the field as a []byte instead of a string.

Reply:
oh ok